[FEATURE] Updated translations from Phrase (PIX-13331)

pix-service-prod opened the pull request:
Exported the most recent locale files from Phrase

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr9468.review.pix.fr)
  * [App (.org)](https://app-pr9468.review.pix.org)
  * [Orga (.fr)](https://orga-pr9468.review.pix.fr)
  * [Orga (.org)](https://orga-pr9468.review.pix.org)
  * [Certif (.fr)](https://certif-pr9468.review.pix.fr)
  * [Certif (.org)](https://certif-pr9468.review.pix.org)
  * [Junior](https://junior-pr9468.review.pix.fr)
  * [Admin](https://admin-pr9468.review.pix.fr)
  * [API](https://api-pr9468.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr9468.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr9468/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr9468/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr9468/environment)

lionelB commented:
@1024pix/team-devcomp il y a des clés de trad à vous qui ont été rajoutées lors de la synchro avec phrase, si jamais c'est ok  on vous laisse mettre une coche

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

AndreiaPena commented:
Nous avions des clés qui s'ajoutait coté phrase que nous avons du supprimer.
Pour Certif c'est ok ✅, je vous laisse re-checker @lionelB et @Faraopix si c'est toujours bon de votre coté.
